lib/measured/rails/validations.rb

```diff
@@ -25,7 +25,7 @@ def validate_each(record, attribute, measurable)
       record.errors.add(attribute, message("is not a valid unit")) unless valid_units.include?(measured_class.conversion.to_unit_name(measurable_unit))
     end
 
-    if measured_class.valid_unit?(measurable_unit)
+    if measured_class.valid_unit?(measurable_unit) && measurable_value.present?
```

Comment:
Hah, I remember looking at this before. The reasoning to keep this was that `measured` won't be accessible outside of `measured-rails` - and app code using `measured-rails` would create valid `Measured` objects. Of course, real life doing its thing here. :+1: